plugins/maxim/README.md

```
@@ -34,24 +34,30 @@ go get github.com/maximhq/bifrost/plugins/maxim
 
 ```bash
 export MAXIM_API_KEY=your_maxim_api_key
+export MAXIM_LOG_REPO_ID=your_maxim_log_repo_id
 ```
 
-2. Setup flags to use the plugin
-   Use include `maxim` in `--plugins` and your maxim log repo id in `--maxim-log-repo-id"`
+2. Setup flags to add the plugin
```

markdownlint-cli2 (0.17.2)

Ordered list item prefix
Expected: 1; Actual: 2; Style: 1/1/1

(MD029, ol-prefix)